### PR TITLE
xignal devices

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
#### Summary

Xignal mousetrap devices

#### Changes

- ...
- ...

#### Release Notes

- ...

### CODE REVIEW

**Comment by Jaime-Trinidad** [comment]
@johanstokking Xignal still don't send decoder and examples

File: vendor/xignal/xignal-mousetrap.yaml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867,
+    # RU864-870
+    profiles:
+      EU863-870:
+        # Optional identifier of the vendor of the profile. When you specify the vendorID, the profile is loaded from
+        # the vendorID's folder. This allows you to reuse profiles from module or LoRaWAN end device stack vendors.
+        # If vendorID is empty, the current vendor ID is used. In this example, the vendorID is the current vendor ID,
+        # which is verbose.
+        vendorID: example
```
Comment (johanstokking) [inline vendor/xignal/xignal-mousetrap.yaml:27]: This doesn't seem right

File: vendor/xignal/xignal-mousetrap.yaml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+# Commercial information
+resellerURLs:
+  - name: 'Find your local Xignal dealer'
+    region:
+      - European Union
+      - United States
+    url: https://dimosystems.com/dealers/
+msrp:
+  EUR: 9999
+  USD: 9999
```
Comment (johanstokking) [inline vendor/xignal/xignal-mousetrap.yaml:119]: This neither

**Review by johanstokking — COMMENTED** [review]
Is this draft?

**Comment by Jaime-Trinidad** [comment]
> Is this draft?

Kind,  I'm supporting a manager of Dimo System because they don't have technical people for the moment, when they have time they will send decoder and examples, I fix the files and change some information.

**Comment by Jaime-Trinidad** [comment]
@kschiffer I make changes to the files as @johanstokking  asked, also I added decoder and codec files with examples, already tested.

File: vendor/xignal/mousetrap.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      data.trapState = 'wakeup';
+    }
+    data.msgId = input.bytes[0];
+    data.battVoltage = input.bytes[1]/10;
+    data.temperature = (input.bytes[2] << 8 | input.bytes[3]) / 100;
+    data.id = input.bytes[10] << 8 | input.bytes[5];
+       
+        
+    
+
```
Comment (kschiffer) [inline vendor/xignal/mousetrap.js:29]: Remove excess whitespace.

**Review by kschiffer — APPROVED** [review]
Not sure if I'm the right person to review this. I can only do a sanity check.

I guess as long as the syntax is correct this should be good enough for now.

**Pull request merged**